### DISCUSSION TITLE
Errors during the `azd init` - an issue with az installation

### DISCUSSION THREAD

**Discussion by NikiforovAll** [Azure/azure-dev#439] [Q&A] [answered]
Hi,

I encountered the issue with `azd` usage. It looks like I'm lacking `az` installed, but I have it installed on my machine. Any ideas on how to fix it?

![image](https://user-images.githubusercontent.com/8037439/183285032-90ed6a2a-5f73-4897-a9a9-898920185d5b.png)

### Environment:

Windows Terminal + GitBash

```bash
/d/dev/azd-cosmos-todo
$ where azd
C:\Users\Oleksii_Nikiforov\AppData\Local\Programs\Azure Dev CLI\azd.exe
/d/dev/azd-cosmos-todo
$ where az
C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin\az
C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin\az.cmd
/d/dev/azd-cosmos-todo
$ az version
Azure-cli    Azure-cli-core    Azure-cli-telemetry
-----------  ----------------  ---------------------
2.39.0       2.39.0            1.0.6
/d/dev/azd-cosmos-todo
$ azd version
azd version 0.1.0-beta.3 (commit 77ab01c3bcf40d12852d543506ecbb935a779f81)
/d/dev/azd-cosmos-todo
$ echo $SHELL
/usr/bin/bash
```

**Comment by karolz-ms** [comment, marked as answer]
Oleksii, I believe this is a bug; I have created an issue to have it fixed: https://github.com/Azure/azure-dev/issues/441

Thanks again for reporting this.
